Dockerfile.client-local

```diff
@@ -8,7 +8,7 @@ COPY app/client /app
 RUN npm run build
 
 # production environment
-FROM nginx:alpine
+FROM nginx:1.23.1-alpine
```

Comment:
We should probably make the same change for Dockerfile.client, that's the one that get's deployed publicly.